src/Routes/AddressRoute.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tychovbh\Mvc\Routes;
+
+class AddressRoute extends AbstractRoutes implements Routes
+{
+    /**
+     * @param array $options
+     */
+    public static function routes(array $options = [])
+    {
+        $instance = self::instance();
+        $instance->index('addresses', $options);
+        $instance->show('addresses', $options);
+        $instance->store('addresses', $options);
```

Comment:
add validation to middleware 
```php
$instance->store('addresses', $options, ['validate']);
```
this will automatically make the request look for a formrequest with this logic:
```php
$request_method = 'store';
$request_name = 'addresses';
$form_request_class = ucfirst($request_method) . Str::singular($request_name); // StoreAddress
$form_request_instance = $form_request_class();
$form_request_instance->validate();
```